src/shared/components/Resources/ResourceEditor.tsx

```diff
@@ -68,7 +70,14 @@ const ResourceEditor: React.FunctionComponent<ResourceEditorProps> = props => {
           )}
         </div>
         <div className="controls">
-          {isEditing && valid && (
+          {!isEditing && valid && (
+            <Switch
+              checkedChildren="expanded"
+              unCheckedChildren="expand"
```

Comment:
to test as well?

Reply:
added tests in 759c033fceb7fcf8ad141fea41c50e61ee195945